software/desktop-environments/en.md

```diff
@@ -30,4 +30,26 @@ Solus offers the MATE desktop environment / experience, tailored to advanced use
 
 ``` bash
 sudo eopkg install -c desktop.mate
-```
\ No newline at end of file
+```
+
+## Plasma
+
+Solus will offer the KDE Plasma desktop environment / experience in the next comming release. Plasma is a powerful and highly customizable desktop enviroment built with Qt and the KDE framework. You can try it right now by running the following command:
+
+``` bash
+sudo eopkg install -c desktop.kde
+```
+
+## i3
+
+Solus offers i3 as a tiling window manager for advanced users that seek window manager instead of a full desktop environment. i3 uses keyboard shortcuts to split windows either horizontally or vertically. Users wanting to try i3 can install it by running the following command:
+
+``` bash
+sudo eopkg install -c desktop.i3
```

Comment:
This isn't the correct process, you only need to install the i3 package.

Reply:
Done with https://github.com/solus-project/help-center-docs/pull/163/commits/0282f763b9e574d052f6178e5ff2caccddf4ad77